File: Code Boiler/Dart/Wallet.dart
class Wallet {
  int balance = 0;

  void viewBalance() {
    print("Your balance is: $balance");
  }

  void deposit(int amount) {
    balance += amount;
    print("$amount has been added to your account. Your balance is: $balance");
  }

  static void main() {
    Wallet wallet = Wallet();
    wallet.viewBalance();
    wallet.deposit(100);
    wallet.viewBalance();
  }
}

File: Code Boiler/python/wallet.py
# WALLET PLATFORM - PYTHON 


class Wallet: 
    def __init__(self, balance):
        self.balance = 0 

    def view_balance(self):
        print("Your balance is: ", self.balance) 


    def deposit(self, amount): 
        self.balance += amount
        print(f"{amount} has been added to your account. Your new balance is: {self.balance}")

wallet = Wallet() 
wallet.view_balance() 
wallet.add_money(50) 
wallet.view_balance()  